cst438_gradebook/src/main/java/com/cst438/domain/Assignment.java

```diff
@@ -13,22 +14,29 @@
 
 @Entity
 public class Assignment {
-	
+
 	@Id
 	@GeneratedValue(strategy=GenerationType.IDENTITY)
 	private int id;
-	
+
 	@ManyToOne
 	@JoinColumn(name="course_id")
 	private Course course;
-	
+
 	@OneToMany(mappedBy="assignment")
 	private List<AssignmentGrade> assignmentGrades;
-	
+
 	private String name;
 	private Date dueDate;
 	private int needsGrading;  // 0 = false,  1= true (past due date and not all students have grades)
-	
+
+	public Assignment(){}
+	public Assignment(Course course, String name, Date dueDate, int needsGrading) {
```

Comment:
You can also add like missing assignment

Reply:
I'm not sure I understand this one 🤔